components/RoadmapPage/RoadmapPage.tsx

```diff
@@ -0,0 +1,146 @@
+import Swiper, { EffectCreative, Mousewheel } from "swiper"
+import { useMediaQuery } from "usehooks-ts"
+import { useEffect, useState } from "react"
+import Slider from "../../shared/Slider"
+import Stage from "./Stage"
+import Layout from "../Layout"
+import { StageData } from "./types"
+import data from "./data.json"
+
+Swiper.use([Mousewheel])
+
+const RoadmapPage = () => {
+  let offset
```

Comment:
can you move this declaration inside your `useEffect` hook?

It's only being used there, correct?